components/brave_extension/extension/brave_extension/elementPicker.html

```diff
@@ -0,0 +1,108 @@
+<head>
+  <meta charset="utf-8">
+  <title>Brave Element Picker</title>
+  <style>
```

Comment:
okay so i really tried to include that `poppins.css` file but i was not able to pull it in. since it lives at `chrome://` and the extension lives in `chrome-extension://` it denies it because the extension is accessing an internal resource. i wasn't able to include it as a web-accessible-resource either. for now i just did `font-family: "Noto Sans", Arial, sans-serif;` because i saw that style on the `brave://adblock` page too.

Reply:
ah, sorry about that! If I knew it was that difficult to get working I wouldn't have even suggested it :sweat: